unifi/Dockerfile

```diff
@@ -9,15 +9,15 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN \
     apt-get update \
     && apt-get install -y --no-install-recommends \
-        binutils=2.34-6ubuntu1.4 \
+        binutils=2.34-6ubuntu1.6 \
```

Comment:
There are multiple things being changed in this PR, which should have been separate PRs.

../Frenck

Reply:
without this versions build crash because old packages are gone.
binutils=2.34-6ubuntu1.6
openjdk-11-jdk-headless=11.0.18+10-0ubuntu1~20.04.1

Reply:
@sinclairpaul no, because without: 
binutils=2.34-6ubuntu1.6
openjdk-11-jdk-headless=11.0.18+10-0ubuntu1~20.04.1

build crash because old packages are gone.

Reply:
https://github.com/Wolvverine/addon-unifi/actions/runs/5361029255/jobs/9726590936

Reply:
> without this versions build crash because old packages are gone.

Sure, I understand that, that wasn't the feedback.

../Frenck